cran-comments.md

```diff
@@ -0,0 +1,18 @@
+## R CMD check results
+
+0 errors | 0 warnings | 2 notes
+
+* checking CRAN incoming feasibility ... [3s/19s] NOTE
+  Maintainer: ‘Jacob Kastl <jacob.kastl@gmail.com>’
+  
+  New submission
+
+* checking for future file timestamps ... NOTE
+  unable to verify current time
```

Comment:
This is a weird one that pops up sometimes, I don't know why or how to avoid it...

Reply:
yea seems to be related to this: https://stackoverflow.com/questions/63613301/r-cmd-check-note-unable-to-verify-current-time

Not sure what to do about it. It does not show upo in the logs when running `devtools::check_win_devel()`, see https://win-builder.r-project.org/mjZZT7ehT2C0/